enterprise-suite/Makefile

```diff
@@ -84,7 +84,15 @@ delete-local:  ## Delete chart from cluster with helm
 	-helm delete --purge $(RELEASE_NAME)
 
 install-local: $(HELM_CHARTS_DIR)/docs/$(RELEASE).tgz install-helm delete-local  ## Install local chart
-	helm install $(HELM_CHARTS_DIR)/docs/$(RELEASE).tgz --name=$(RELEASE_NAME) --namespace=$(NAMESPACE) --debug --wait
+	@if test -z "$(BINTRAY_USERNAME)"; then \
+		echo BINTRAY_USERNAME required \
+	fi
+	@if test -z "$(BINTRAY_PASSWORD)"; then \
+		echo BINTRAY_PASSWORD required \
+	fi
+	echo "Installing helm charts"
+	@helm install $(HELM_CHARTS_DIR)/docs/$(RELEASE).tgz --name=$(RELEASE_NAME) --namespace=$(NAMESPACE) --debug --wait \
```

Comment:
Does this have the leading '@' to prevent leakage when `imageCredentials.password=$(BINTRAY_PASSWORD)` is printed?  Otherwise, I think it's nice to see this...

Reply:
Yes that's why.